docs/RELEASE.MD

```diff
@@ -0,0 +1,62 @@
+# Release
+
+The purpose of this document is to describe the procedure to release this library.
+
+## Release steps
+
+1. Create a new local Git branch.
+2. In [daml.yaml](../daml.yaml), update the `version` value as required :
+   - To check what `versions` already exist, check [here](https://github.com/digital-asset/daml-ctl/tags).
+   - Note - do not add a `v` in front of the `version` in [daml.yaml](../daml.yaml). The build script handles the naming convensions of the release, the tag and the release artifacts.
+3. Run `daml build` :
+   - This command validates the specified `version`;
+   - A successful build means the `version` specified is valid.
+4. Push your branch to GitHub.
+5. Create a pull request to merge your branch to `master`.
+6. Get approval and merge this branch.
+7. When the branch gets merged to `master`, a CI job gets triggered which will build the project and then preform the release:
```

Comment:
typo: preform -> perform

Reply:
Sorry, I should have proof read it before committing.